transport/rif_comms/client.py

```diff
@@ -13,38 +13,49 @@
 
 
 class Client:
+
     """
     Class to connect and operate against a RIF Communications pub-sub node.
     """
 
-    def __init__(self, rsk_address: Address, grpc_api_endpoint: str):
+    def __init__(self, rsk_address: Address,
+                 grpc_api_endpoint: str,
+                 grpc_client_timeout: float = 30):
```

Comment:
i think this should be `grpc_client_timeout: float = DEFAULT_RIF_COMMS_GRPC_CLIENT_TIMEOUT_SECONDS`